src/docs/architecture.md

```diff
@@ -60,4 +62,5 @@ graph TD
 ![ECS](../public/QC%20Architecture-ECS.jpg)
 
 **VM Infrastructure**
-![VM](../public/QC%20Architecture-VM.jpg)
\ No newline at end of file
+![VM](../public/QC%20Architecture-VM.jpg)
+
```

Comment:
This page isn't rendering correctly for me 🤔 

<img width="865" alt="Screenshot 2025-05-22 at 3 26 15 PM" src="https://github.com/user-attachments/assets/8b353979-7239-4933-9a20-87bdb227d27f" />

Reply:
Currently working on resolving this!